addons/maintenance/tests/test_calendar_with_recurrence.py

```diff
@@ -23,26 +24,30 @@ def test_dblclick_event_from_calendar(self):
             'equipment_id': equipment.id,  # necessary for the tour to work with mrp_maintenance installed
             'maintenance_type': 'preventive',
             'recurring_maintenance': True,
+            'repeat_until': datetime.combine(date.today(), datetime.min.time()) + timedelta(days=8, hours=10),
             'repeat_interval': 1,
             'repeat_unit': 'day',
             'duration': 1,
         })
         self.env['maintenance.request'].create({
```

Comment:
Batch creation seems impossible because of this: https://github.com/odoo/odoo/blob/d022aff3a316cbb40816f74510aeb45243ce84a4/addons/maintenance/models/maintenance.py#L370 https://github.com/odoo/odoo/blob/d022aff3a316cbb40816f74510aeb45243ce84a4/addons/maintenance/models/maintenance.py#L353-L354

Am I allowed to replace `self` with `request` or should I separate creates again?

Reply:
Yes :ok_hand: